introduction/introduction.md

```diff
@@ -0,0 +1,15 @@
+This file holds the text to accompany the [introduction](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321606) video.
+This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).
+
+# Introduction
+
+This Learning Path gives an introduction to inner source.
+Inner source is the application of open source practices and principles to software development within the enterprise.
```

Comment:
Maybe a slight rewording? Something like:

> Inner source is the application of open source practices and principles to proprietary or closed source software development.

As is, it reads to me like the target audience is for-profit companies (enterprises). There's no reason why inner sourcing practices can't be employed elsewhere too (non-profits, etc).

Reply:
I agree with the comment Demian is making about nonprofits etc but I don't think the word "enterprise" is off putting or confusing, but I do think the term "closed source software development" is confusing. As a reader of the original phrasing, I can infer the meaning of inner source and expand that to see its relevance in any organization.

Reply:
Yeah that's fair. Perhaps I was being a tad too pedantic there :)